### ISSUE TITLE
two commits highlighted when left mouse button clicked and held over commit list.

### ISSUE THREAD

**Issue by jensenr30** [Murmele/Gittyup#699]
This bug seems pretty simple;  I could probably fix it.  Feel free to assign to me.

Steps to reproduce bug:
- left click (and hold) on a commit in the list of commits
- drag the mouse up or down past a few other commits
- release left mouse button

At this point, two commits will be highlighted.  See attached video.

I originally thought this was a bug, but maybe it could be turned into a feature!  It would be pretty nifty to make the light-blue highlighter continue to follow the cursor around regardless of whether the mouse button is held down or not.  It helps me visually the eye line up commit circles, commit messages, and the author/date.

Thoughts?

P.S.
Pay no attention to all the "gitslam!" commits in my repo in the video - its just a personal repo that no one else uses.

[tmp.webm](https://github.com/Murmele/Gittyup/assets/3083835/f302dbd5-7ed9-4acf-8a61-531db27f58ef)

**Comment by Murmele** [comment]
Yes this could be interesting, it is confusing that it stops highlighting another commit just when released
